packages/client/src/App.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+// import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
-import { LandingPage } from './containers/LandingPage/LandingPage.Container';
-import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
+import Login from './component/Login';
+// import { LandingPage } from './containers/LandingPage/LandingPage.Container';
+// import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
 
 function App() {
   return (
     <div className="app">
-      <Router>
+      <Login />
+      Hello
+      {/* <Router>
```

Comment:
I would advice to use the routes from the get go, do not comment them out, so you could now just go and use your Login component for test (an maybe later too) in the LandingPage container.

Reply:
Exactly, I second @shpomp 😄